src/Buzzer.cpp

```diff
@@ -1,44 +1,83 @@
-/*
-  Photoresistor.h - Simple Library for reading from a Photoresistor
+/**
+  Calico Home Security System, Buzzer.cpp
+  Purpose: sets the tone of the Buzzer and if it's on and off
+
+  @author Chris Nash, Jason Bruderer, David Tille, Tyler Jacobs
+  @version To be Determined
 */
 #include "Arduino.h"
 #include "Buzzer.h"
-
+/**
+  Constructor sets the pin of the Buzzer and componentType to Buzzer
+*/
 Buzzer::Buzzer(int pin) : Component(pin, "BUZZER"){
   pinMode(pin, OUTPUT);
 }
-
+/**
+  Base constructor
+*/
 Buzzer::Buzzer(){
 }
-
+/**
+  Uses the int frequency to determine the pitch of the tone being passed to the
+    tone function.
+    This version of the soundTone Function does not use duration with the tone
+    function.
+*/
 void Buzzer::soundTone(int frequency) {
+  //Need to take a look at while we're using properies for a Serail.println
+  //does not affected the tone
   if(Properties::DEBUGGING_ACTIVE) Serial.println("Buzzer has started sounding");
   tone(this->getPin(), frequency);
 }
```

Comment:
I think this may have been one that I forgot to switch over to the Logger::Log paradigm. I don't know if we really need to log this anyway, but I guess we can. Maybe at the Trace log level.